packages/cli/test/cases/build.config.error-workspace-absolute/build.config.error-workspace-absolute.spec.js

```diff
@@ -1,18 +1,18 @@
 /*
  * Use Case
  * Run Greenwood build command with a bad value for workspace directory (that doesn't exist) in a custom config.
- * 
+ *
  * User Result
  * Should throw an error.
- * 
+ *
  * User Command
  * greenwood build
- * 
+ *
  * User Config
  * {
  *   workspace: path.join(__dirname, 'noop')
  * }
- * 
+ *
```

Comment:
are these coming from your IDE by chance?  It might be good to turn that feature off (for this project anyway).  All the formatting is unfortunately adding a lot a noise to this PR.

Reply:
Yea, but what's the point of having spaces there anyway?

Reply:
hmm... that wasn't really the concern though.

> All the formatting is unfortunately adding a lot a noise to this PR.

Reply:
two birds with one stone?

Reply:
Again, that wasn't really my concern.  I will try and explain on the call instead if we have time.

Reply:
So we didn't get to it on the call, but just to emphasize the point of focus was on the _diff_ introduced by the formatting changes, regardless of it was an addition / removal.  

In general (and yes, maybe we have been lax on formatting, maybe we should add [prettier](https://prettier.io/) to the project?) but in the face of an important PR like this (bug, large feature or refactoring) formatting changes purely for the sake of formatting changes are discouraged.  It adds "noise" which is that changes presented are not actually changes to have to worry about, yet it still counts as extra files and changes to the diff that get in the way of what is really important and what a reviewer would _want_ to read, is all.  This PR would be at least few files less and definitely many lines less if just the formatting was removed, which I think speaks for itself.

Always up for consistent formatting, and as a standalone PR that is definitely doable.  So just to politely ask please turn off any auto formatting (for this project) in your IDE so these kinds of changes don't have to keep happening.  Thanks!!! ✌️